src/components/ArticleBlockSmall.jsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from "react-router-dom";
+
+function ArticleBlockSmall({ article }) {
+  const date = new Date(article.created_at);
+  const format = { year: "numeric", month: "long", day: "numeric" };
+  const formattedDate = date.toLocaleDateString("en-US", format);
+
+  return (
+    <>
+      <li className="article-block-small">
+        <img src={article.article_img_url} alt={article.title} />
```

Comment:
consider what other details about the article the user might want to know before they click on the article :)